src/aiosmtplib/connection.py

```diff
@@ -248,9 +248,9 @@ def source_address(self) -> str:
                     host=self.hostname, port=self.port, err=err
                 )
             )
-        except asyncio.TimeoutError as exc:
+        except asyncio.TimeoutError:
             self.close()
-            raise SMTPTimeoutError(str(exc))
+            raise SMTPTimeoutError("Connect timeout error")
```

Comment:
Can you make a new exception `SMTPConnectTimeoutError` that inherits from `SMTPConnectError` and `SMTPTimeoutError`? That should make handling this error a lot easier.

Reply:
Good idea. Initially, I had a similar thought, but I decided not to over-complicate at the beginning. Done